pkg/features/kube_features.go

```diff
@@ -483,6 +483,12 @@ const (
 	// Disables Accelerator Metrics Collected by Kubelet
 	DisableAcceleratorUsageMetrics featuregate.Feature = "DisableAcceleratorUsageMetrics"
 
+	// owner: @pacoxu
+	// alpha: v1.24
+	//
+	// Disables Cadvisor Process Metrics Collected by Kubelet
+	DisableCadvisorProcessMetrics featuregate.Feature = "DisableCadvisorProcessMetrics"
```

Comment:
Feature gates are meant to be temporary in order to transition from one state (metrics always enabled) to the opposite (metrics always disabled).  I don't think there are plans to permanently disable process metrics, so this should be done in kubelet configuration, rather than as a feature gate, if this is the direction sig-node wants to take.

Reply:
I also thought about that. 
I opened #107660 to use a  flag.

A reason that I use a feature gate is that there is an example that is beta but defaults false for special reasons. 
I just wonder which is a more flexible way to implement this. 
https://github.com/kubernetes/kubernetes/blob/9d2b361ebc7ef28f7cb75596ef40b7c239732d37/pkg/features/kube_features.go#L44-L50